### PR TITLE
Cfm data viz integration

### PR DESCRIPTION
**Pull request by crfmc**
Associated with [Trello ticket 808](https://trello.com/c/TeLOTr1O/808-d3-visualizations-for-smaht-portal)

- Alluvial and Consortium Map visualizations with table views
- Creates two new static sections (consortium-map and alluvial)
- Adds consortium-map to `/about/consortium/awardees`
- Adds alluvial to `/about/consortium/data`
- Adds [topoJSON-client](https://github.com/topojson/topojson-client) library

### CODE REVIEW

**Review by Bianca-Morris — CHANGES_REQUESTED** [review]
Nice work on this! I think you're 99% of the way there... Just a few small details before we call this done:

- Table for the consortium map is overflowing the page. Would be good to get that to have a max of the page width, and have it scroll. (Should be as simple as slapping `.table-responsive` on it.) In the future, we might want to have it default to the table view on mobile devices, or add a two-finger zoom feature to the map, but for now, that should be good enough.
- A few other things in comments

Comments that don't block merge:
- Thinking there might be a way to use one SVG of the map-marker, and then use CSS or D3 to change the fill programmatically. Not a huge issue, but would reduce the number of resources required to load in future.
- I did end up adding a `consortia` and `submission_centers` key/value pair to each of your Static Section inserts when I merged in `ui-dev`. Apparently public/shared pages do not work as expected when this value is not defined. Just a note for the future!

File: src/encoded/tests/data/master-inserts/static_section.json [hunk 1]
```diff
@@ -24,5 +24,31 @@
         },
         "section_type": "Page Section",
         "uuid": "3fd92b15-9771-4218-a3f9-77b15805443e"
+    },
+    {
+        "name": "consortium-map",
+        "title": "Consortium Map",
```
[the remark this answers is not in the document]
Comment (Bianca-Morris) [reply src/encoded/tests/data/master-inserts/static_section.json:42]: Specifically the title, by the way. Name can stay the same/is just used for the anchor ID.

File: package.json [hunk 2]
```diff
@@ -64,12 +64,11 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.15.4",
-    "@hms-dbmi-bgm/react-workflow-viz": "0.1.5",
```
[the remark this answers is not in the document]
Comment (crfmc) [reply package.json:67]: Hmm I'm actually not sure when/why this was removed. I'm going to add this back and see if it affects d3.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -7,6 +7,17 @@ smaht-portal
...
+* Alluvial and Consortium Map visualizations with table views
+* Creates two new static sections (consortium-map and alluvial)
+* Adds consortium-map to `/about/consortium/awardees`
+* Adds alluvial to `/about/consortium/data`
+* Adds [topoJSON-client](https://github.com/topojson/topojson-client) library
+
+
 0.4.0
 =====
 `PR 19: UI-Dev <https://github.com/smaht-dac/smaht-portal/pull/20>`_
```
[the remark this answers is not in the document]
Comment (crfmc) [reply CHANGELOG.rst]: Removed. Thanks for explaining!

File: src/encoded/static/scss/encoded/modules/_static-pages.scss [hunk 4]
```diff
@@ -591,4 +591,700 @@ body[data-path="/"]>#application>#layout {
...
+    
+    }
+    
+}
+
+
+.popover {
+    font-family: Inter;
+    max-width: 400px;
+
```
[the remark this answers is not in the document]
Comment (crfmc) [reply src/encoded/static/scss/encoded/modules/_static-pages.scss]: Added. Thanks!

File: src/encoded/tests/data/master-inserts/static_section.json [hunk 5]
```diff
@@ -24,5 +24,31 @@
...
+            "filetype": "jsx",
+            "collapsible": false,
+            "default_open": true
+        },
+        "section_type": "Page Section",
+        "uuid": "f1601ba5-483b-4133-a76b-5bec7caa2dd2"
+    },
+    {
+        "name": "alluvial",
+        "title": "Alluvial",
```
[the remark this answers is not in the document]
Comment (crfmc) [reply src/encoded/tests/data/master-inserts/static_section.json]: Adjusted. Thank you!

**Review by Bianca-Morris — APPROVED** [review]

**Pull request merged**